packages/stylelint-plugin/src/stylelintPlugin.spec.ts

```diff
@@ -537,57 +537,57 @@ const testExamples: TestExample[] = [
     language: "scss",
     testName: "transitive kaizen tokens are fixed",
     input:
-      '@import "~@kaizen/design-tokens/sass/color-vars"; $foo: $kz-var-color-wisteria-800; .foo { color: $foo; }',
+      '@import "~@kaizen/design-tokens/sass/color"; $foo: $color-purple-800; .foo { color: $foo; }',
     expectedOutput:
-      '@import "~@kaizen/design-tokens/sass/color"; @import "~@kaizen/design-tokens/sass/color-vars"; $foo: $kz-var-color-wisteria-800; .foo { color: $color-purple-800; }',
-    expectedWarnings: 1,
+      '@import "~@kaizen/design-tokens/sass/color"; $foo: $color-purple-800; .foo { color: $color-purple-800; }',
+    expectedWarnings: 0,
   },
   {
     language: "scss",
     testName: "transitive kaizen tokens containing multiple values are fixed",
     input:
-      '@import "~@kaizen/design-tokens/sass/color-vars"; $foo: $kz-var-color-wisteria-800 $kz-var-color-wisteria-700 $kz-var-color-wisteria-800; .foo { color: $foo; }',
+      '@import "~@kaizen/design-tokens/sass/color"; $foo: $color-purple-800 $color-purple-700 $color-purple-800; .foo { color: $foo; }',
     expectedOutput:
-      '@import "~@kaizen/design-tokens/sass/color"; @import "~@kaizen/design-tokens/sass/color-vars"; $foo: $kz-var-color-wisteria-800 $kz-var-color-wisteria-700 $kz-var-color-wisteria-800; .foo { color: $color-purple-800 $color-purple-700 $color-purple-800; }',
-    expectedWarnings: 3,
+      '@import "~@kaizen/design-tokens/sass/color"; $foo: $color-purple-800 $color-purple-700 $color-purple-800; .foo { color: $color-purple-800 $color-purple-700 $color-purple-800; }',
+    expectedWarnings: 0,
   },
   {
     language: "scss",
     testName:
-      "color manipulation functions are warned about, even when a transitive kaizen token is used",
+      "color manipulation functions are not warned about (SASS compiler is responsible for them), even when a transitive kaizen token is used",
     input: `
-      @import "~@kaizen/design-tokens/sass/color-vars";
-      $foo: $kz-var-color-wisteria-800;
+      @import "~@kaizen/design-tokens/sass/color";
+      $foo: $color-purple-800;
       .foo {
         color: mix($foo, $white, 80%);
       }`,
     expectedOutput: `
-      @import "~@kaizen/design-tokens/sass/color-vars";
-      $foo: $kz-var-color-wisteria-800;
+      @import "~@kaizen/design-tokens/sass/color";
+      $foo: $color-purple-800;
       .foo {
-        color: mix($kz-var-color-wisteria-800, $white, 80%);
+        color: mix($color-purple-800, $white, 80%);
       }`,
-    expectedWarnings: 3,
+    expectedWarnings: 1,
```

Comment:
One thing I struggle with in this test suite... what's the 1 warning I'm expecting?

What are the two I'm no longer expecting? It's hard to know what behaviour change is expected by this diff